cmake/modules/FindMayaUsd.cmake

```diff
@@ -0,0 +1,90 @@
+#
+# Simple module to find MayaUsd.
+#
+# This module searches for a valid MayaUsd installation.
+# It searches for MayaUsd's libraries and include header files.
+#
+# Variables that will be defined:
+# MAYAUSD_FOUND           Defined if a MayaUsd installation has been detected
+# MAYAUSD_LIBRARY_DIR     Path to MayaUsd libraries directory
+# MAYAUSD_INCLUDE_DIR     Path to the MayaUsd include directory
+# MAYAUSDAPI_LIBRARY      Path to MayaUsdAPI library
+# MAYAUSD_MOD_PATH        Path to mayaUSD.mod file
+#
+
+find_path(MAYAUSD_INCLUDE_DIR
+        mayaUsd/mayaUsd.h
+    HINTS
+        ${MAYAUSD_LOCATION}
+    PATH_SUFFIXES
+        include/
+    DOC
+        "MayaUsd header path"
+)
+
+find_path(MAYAUSD_LIBRARY_DIR
+        mayaUsd.lib
+    HINTS
+        ${MAYAUSD_LOCATION}
+    PATH_SUFFIXES
+        lib/
+    DOC
+        "MayaUsd libraries path"
+)
```

Comment:
Why do you need to find this path? You just need the API one (below) - right? If you do need this one, then you'll have to adjust for linux/mac because the extension is not .lib on those platforms.